streams/src/test/java/org/apache/kafka/streams/integration/QueryableStateIntegrationTest.java

```diff
@@ -93,6 +93,7 @@
     private static final long WINDOW_SIZE = TimeUnit.MILLISECONDS.convert(2, TimeUnit.DAYS);
     private static final int NUM_PARTITIONS = 2;
     private static final int NUM_REPLICAS = NUM_BROKERS;
+    private static final int MAX_WAIT_MS = Integer.getInteger("kafka.test.max.wait.ms", 60000);
```

Comment:
It's an attempt to help you solving random test failures, which I see from time to time. As I understand, you have a large "park" of build-test servers with rather different configuration, so configurable timeouts may be aid with this. In the future same variable can be used in other tests as well.

But still I believe that it's a decision of maintainer whether the proposed update suits the overall solution. So waiting for your suggestions.